snapcraft/cli/env.py

```diff
@@ -28,31 +28,32 @@ class BuilderEnvironmentConfig:
 
     Valid values are:
 
-    - host: the host will drive the build.
```

Comment:
Above, the docstring still includes "If it is not set, a value of `host` is assumed."